.github/workflows/build.yml

```
@@ -28,6 +28,8 @@ jobs:
           yarn
           yarn sass
           yarn build
+          # Prevent GitHub from trying to use Jekyll
+          touh _site/.nojekyll
```

was this meant to be `touch`?

```suggestion
          touch _site/.nojekyll
```